### DISCUSSION TITLE
.css import aren't working

### DISCUSSION THREAD

**Discussion by DennisWarchocki** [neuefische/questions#216] [Web] [answered]
Hello everyone, 
Currently I am trying to give my new feature a styling. I have the styling of the new feature in the form of a .css file I tried to import the.css file in the right place, but nothing happens.

Import + structure:
![Bildschirmfoto 2022-10-05 um 14 33 01](https://user-images.githubusercontent.com/109968361/194061272-a92efdf2-d211-4baa-85a7-10a9f3e94995.png)

this is my css file:
![Bildschirmfoto 2022-10-05 um 14 36 56](https://user-images.githubusercontent.com/109968361/194061803-a0355e9d-51a3-485b-b005-18deb0b31d6f.png)

This is what my feature looks like:
![Bildschirmfoto 2022-10-05 um 14 35 10](https://user-images.githubusercontent.com/109968361/194061577-d23e6378-65fc-4389-9d50-aa33b434fe90.png)

This is how my feature should look like:
![Bildschirmfoto 2022-10-05 um 14 38 13](https://user-images.githubusercontent.com/109968361/194062049-6eb6ec74-fb49-4b1e-a109-6313f10e4554.png)

My Repo:
https://github.com/DennisWarchocki/capstone-project

I would be very happy about support, thank you very much!

**Comment by pixelass** [comment, marked as answer]
Why are you not using styled-components to style your elements?

Can you send the link to the PR/Branch?

**Reply by DennisWarchocki** [reply]
Branch:
https://github.com/DennisWarchocki/capstone-project/tree/timer

> Why are you not using styled-components to style your elements?

Because I just wanted to see the results fast and change it to styled-components later.
I wonder why it's not working with .css because I think it should work.

**Reply by pixelass** [reply]
This is the file where you should import the css file: https://github.com/DennisWarchocki/capstone-project/blob/timer/src/components/CookingTimer.js

**Reply by pixelass** [reply]
In this file, everything is commented, therefore this file cannot work: https://github.com/DennisWarchocki/capstone-project/blob/timer/src/pages/Timer.js

Sidenote: all pages should use lowercase characters. instead of `pages/Timer.js` ist should be `pages/timer.js` since URLs are case insensitive.

**Reply by DennisWarchocki** [reply]
Thanks for the further advice pixel, I'll try to fix that.
